2025-03-22 14:54:49 CST [1]: [3-1] user=,db=,app=,client= LOG:  starting PostgreSQL 15.12 on x86_64-pc-linux-musl, compiled by gcc (Alpine 14.2.0) 14.2.0, 64-bit
2025-03-22 14:54:49 CST [1]: [4-1] user=,db=,app=,client= LOG:  listening on IPv4 address "0.0.0.0", port 5432
2025-03-22 14:54:49 CST [1]: [5-1] user=,db=,app=,client= LOG:  listening on IPv6 address "::", port 5432
2025-03-22 14:54:50 CST [1]: [6-1] user=,db=,app=,client= LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2025-03-22 14:54:51 CST [30]: [1-1] user=,db=,app=,client= LOG:  database system was interrupted; last known up at 2025-03-22 12:55:58 CST
2025-03-22 14:55:02 CST [30]: [2-1] user=,db=,app=,client= LOG:  syncing data directory (fsync), elapsed time: 10.11 s, current path: ./base/1/2607_fsm
2025-03-22 14:55:12 CST [30]: [3-1] user=,db=,app=,client= LOG:  syncing data directory (fsync), elapsed time: 20.11 s, current path: ./base/1/2996
2025-03-22 14:55:22 CST [30]: [4-1] user=,db=,app=,client= LOG:  syncing data directory (fsync), elapsed time: 30.01 s, current path: ./base/1/827
2025-03-22 14:55:32 CST [30]: [5-1] user=,db=,app=,client= LOG:  syncing data directory (fsync), elapsed time: 40.01 s, current path: ./base/16384/3085
2025-03-22 14:55:42 CST [30]: [6-1] user=,db=,app=,client= LOG:  syncing data directory (fsync), elapsed time: 50.00 s, current path: ./base/4/2675
2025-03-22 14:55:52 CST [30]: [7-1] user=,db=,app=,client= LOG:  syncing data directory (fsync), elapsed time: 60.00 s, current path: ./base/5/1255_vm
2025-03-22 14:56:02 CST [30]: [8-1] user=,db=,app=,client= LOG:  syncing data directory (fsync), elapsed time: 70.13 s, current path: ./base/5/3394_fsm
2025-03-22 14:56:12 CST [30]: [9-1] user=,db=,app=,client= LOG:  syncing data directory (fsync), elapsed time: 80.03 s, current path: ./log/postgresql-2025-03-05_164612.log
2025-03-22 14:56:17 CST [30]: [10-1] user=,db=,app=,client= LOG:  database system was not properly shut down; automatic recovery in progress
2025-03-22 14:56:18 CST [30]: [11-1] user=,db=,app=,client= LOG:  redo starts at 0/197D160
2025-03-22 14:56:18 CST [30]: [12-1] user=,db=,app=,client= LOG:  invalid record length at 0/197D248: wanted 24, got 0
2025-03-22 14:56:18 CST [30]: [13-1] user=,db=,app=,client= LOG:  redo done at 0/197D210 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.31 s
2025-03-22 14:56:19 CST [28]: [1-1] user=,db=,app=,client= LOG:  checkpoint starting: end-of-recovery immediate wait
2025-03-22 14:56:24 CST [28]: [2-1] user=,db=,app=,client= LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.896 s, sync=0.437 s, total=5.891 s; sync files=2, longest=0.277 s, average=0.219 s; distance=0 kB, estimate=0 kB
2025-03-22 14:56:27 CST [1]: [7-1] user=,db=,app=,client= LOG:  database system is ready to accept connections
2025-03-22 16:09:49 CST [1]: [8-1] user=,db=,app=,client= LOG:  received fast shutdown request
2025-03-22 16:09:50 CST [1]: [9-1] user=,db=,app=,client= LOG:  aborting any active transactions
2025-03-22 16:09:50 CST [1]: [10-1] user=,db=,app=,client= LOG:  background worker "logical replication launcher" (PID 33) exited with exit code 1
2025-03-22 16:09:50 CST [28]: [3-1] user=,db=,app=,client= LOG:  shutting down
2025-03-22 16:09:50 CST [28]: [4-1] user=,db=,app=,client= LOG:  checkpoint starting: shutdown immediate
2025-03-22 16:09:52 CST [28]: [5-1] user=,db=,app=,client= LOG:  checkpoint complete: wrote 0 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.489 s, sync=0.001 s, total=2.568 s; sync files=0, longest=0.000 s, average=0.000 s; distance=0 kB, estimate=0 kB
2025-03-22 16:09:52 CST [1]: [11-1] user=,db=,app=,client= LOG:  database system is shut down
